demo/js/components/victory-accessibility-demo.js

```diff
@@ -0,0 +1,414 @@
+      { x: 5, y: 65 }
+    ];
+
+    const scatterDataAriaLabels = scatterData.map(data => (
+        "Time " + data.x + " Hour" + " Temperature " + data.y + " Fahrenheit"
+    ));
+
+    const barData = [
+      { x: 1, y: 80 },
+      { x: 2, y: 73 },
+      { x: 3, y: 69 },
+      { x: 4, y: 84 },
+      { x: 5, y: 65 }
+    ];
+
+    const barDataAriaLabels = barData.map(data => (
+      "Time " + data.x + " Hour" + " Temperature " + data.y + " Fahrenheit"
+    ));
+
+    const stackData1 = [
+      {x: "coffee", y: 8},
+      {x: "tea", y: 3},
+      {x: "soda", y: 5}
+    ]
+
+    const stackDataAriaLabels1 = stackData1.map(data => (
+      data.x + " " + data.y
+    ));
+
+    const stackData2 = [
+      {x: "coffee", y: 4},
+      {x: "tea", y: 6},
+      {x: "soda", y: 3}
+    ]
+
+    const stackDataAriaLabels2 = stackData2.map(data => (
+      data.x + " " + data.y
+    ));
+
+    const stackData3 = [
+      {x: "coffee", y: 6},
+      {x: "tea", y: 2}, 
+      {x: "soda", y: 2}
+    ]
+
+    const stackDataAriaLabels3 = stackData3.map(data => (
+      data.x + " " + data.y
+    ));
+
+    const areaData = [
+      { x: 2016, y: 2 },
+      { x: 2017, y: 3 },
+      { x: 2018, y: 5 },
+      { x: 2019, y: 4 },
+      { x: 2020, y: 6 }
+    ];
+
+    const areaDataAriaLabel = () => {
+      const string = areaData.map(data => (
+        data.x + " " + data.y
+      ))
+      return "Area under the data points" + string;
+    }
+
+    const lineData = [
+      { x: "Jan", y: 2 },
+      { x: "Feb", y: 3 },
+      { x: "Mar", y: 5 },
+      { x: "Apr", y: 4 },
+      { x: "May", y: 7 },
+      { x: "Jun", y: 2 },
+      { x: "Jul", y: 3 },
+      { x: "Aug", y: 5 },
+      { x: "Sep", y: 4 },
+      { x: "Oct", y: 7 },
+      { x: "Nov", y: 2 },
+      { x: "Dec", y: 3 }
+    ];
+
+    const lineDataAriaLabel = () => {
+      const string = lineData.map(data => (
+        data.x + " " + data.y
+      ))
+      return "Line chart with the following data points" + string;
+    }
+
+    const groupData1 = [
+      { x: "2013", y: 350 },
+      { x: "2014", y: 300 },
+      { x: "2015", y: 250 },
+      { x: "2016", y: 200 }
+    ];
+
+    const groupData2 = [
+      { x: "2013", y: 150 },
+      { x: "2014", y: 100 },
+      { x: "2015", y: 200 },
+      { x: "2016", y: 180 }
+    ];
+
+    const groupData3 = [
+      { x: "2013", y: 350 },
+      { x: "2014", y: 200 },
+      { x: "2015", y: 150 },
+      { x: "2016", y: 100 }
+    ];
+
+    const groupDataAriaLabels1 = groupData1.map(data => (
+      "In " + data.x + " sales were " + data.y + " dollars."
+    ));
+
+    const groupDataAriaLabels2 = groupData2.map(data => (
+      "In " + data.x + " sales were " + data.y + " dollars."
+    ));
+
+    const groupDataAriaLabels3 = groupData3.map(data => (
+      "In " + data.x + " sales were " + data.y + " dollars."
+    ));
+
+    const AccessibleGroupComponent = ({ ariaLabel, ariaDescribedBy, description, descriptionId, children }) => {
```

Comment:
@boygirl do you have any thoughts on whether I should make this a component? That would make it easier for devs who want to build accessible charts to ensure that when there are groups of paths, that there is a `<desc>` tag rendered in the DOM that describes each path within that group.

Reply:
Apologies for the delay. I'm not opposed to adding an `AccessibleGroupComponent`. It should probably live in `victory-core`